docs/EVENTS.md

```diff
@@ -235,6 +241,8 @@
 | model_save_after | 1.9.4.5 |
 | model_save_before | 1.9.4.5 |
 | model_save_commit_after | 1.9.4.5 |
+| newsletter_send_before | 1.9.4.6 |
+| newsletter_send_after | 1.9.4.6 |
```

Comment:
shold we use the next release number instead of 1.9.4.6?

Reply:
Yeah .... [rjocoleman](https://github.com/rjocoleman) :facepalm:  ... LOL.

Reply:
what do you want the value to be?

Reply:
I guess the next release would be 19.4.17

Reply:
I guess the next release would be 19.4.18

Reply:
Next release should be 19.4.**19**

Reply:
mmmm no :-D last released was .17

Reply:
mhhh ...

![om-vers](https://user-images.githubusercontent.com/5022236/186365541-f77c5011-3501-46ed-ac62-c8502e1b1929.PNG)

Reply:
I changed it to 19.4.18 before I saw the other messages. The last release I see on GH is [19.4.17](https://github.com/OpenMage/magento-lts/releases/tag/v19.4.17) so 19.4.18 is correct for the current 1.9.4.x branch?

Reply:
which is already updated for the next release, the 1.9.4.x is actually the .18 ;-)
I always try to bump the version number right after the release, to be ready for the next release. Otherwise the day of the release we'd be stuck with a PR with only a version bump that we can't get approved and merged.

Reply:
My bad. Raising version number should be last step ....

Reply:
mmm I kinda think the other was around, if you're using the git version you're already using the .18 instead of the .17

but also again i'm having the problem that i tried to help with releasing and I can't do if i don't have a version bump...

Reply:
Its okay as it is. I just did not know/expect that the version was already bumped in Mage.php.

 I think you're right with changing version as soon as possible. (i guess something like `19.4.18-alpha/beta` would be overkill)

Reply:
but then i'd have the same problem the day of release...